docs/design.md

```diff
@@ -0,0 +1,59 @@
+# Enclave-CC Design and Architecture
+
```

Comment:
Spell out CC for people without the context.
```suggestion
Enclave-based Confidential Containers
```

Reply:
Done. For all resolved items, I will simply click "Resolve conversation" without further acknowledge.